CODEOWNERS

```diff
@@ -1 +1,7 @@
 @sigstore/sget-codeowners
+
+# The CODEOWNERS are managed via a GitHub team, but the current list is (in alphabetical order):
```

Comment:
I think this might be a lie, at time of writing, based on what @dlorenc mentioned. But perhaps we could make this adjustment to make it easier for new folks to join on in the future?